apps/andi/assets/css/edit.scss

```diff
@@ -51,5 +98,19 @@
 @include area(metadata-form, level-of-access);
 @include area(metadata-form, language);
 @include area(metadata-form, homepage);
-@include area(metadata-form, cancel-btn);
-@include area(metadata-form, save-btn);
+
+@include area(edit-button-group, cancel-btn);
+@include area(edit-button-group, save-btn);
+@include area(edit-button-group, test-btn);
+
+@include area(url-form, source-url);
+@include area(url-form, test-section);
+
+
+.box-header {
```

Comment:
This belongs in the app.scss file if it's meant to be generic, otherwise I would make it an element of the edit-page block (`edit-page__box-header`)

Reply:
Not sure how BEM works in this case. It's a modifier to a "generic" element that's only ever part of a form group.

Reply:
If it's only ever going to be part of the form group I would do `edit-page__box-header`